go/sqltypes/type.go

```diff
@@ -159,6 +159,7 @@ const (
 	HexNum     = querypb.Type_HEXNUM
 	HexVal     = querypb.Type_HEXVAL
 	Tuple      = querypb.Type_TUPLE
+	BitNum     = querypb.Type_BITNUM
```

Comment:
Don't we need BitVal too?

Reply:
we can just convert it to BitNum, do not see a point in keeping two representation.